3_javascript/javascript1/week1/main.js

```diff
@@ -4,3 +4,18 @@ let yearFuture = 2045;
 let age = yearFuture - yearOfBirth;
 
 console.log("You will be " + age + " years old in " + yearFuture)
+
+
+//Goodboy-Oldboy
+let dogYearOfBirth = 2023;
+let dogYearFuture = 2027;
+let dogYear = 7;
+let shouldShowResultInDogYears = true;
+let dogAge = dogYearFuture - dogYearOfBirth;
+
+if (shouldShowResultInDogYears === true){
```

Comment:
you do not need to declare in an if statement that `shouldShowResultInDogYears === true`, simply will be enough to write :`if (shouldShowResultInDogYears ){...` because you declared above it.